src/sentry/eventstore/reprocessing/redis.py

```diff
@@ -0,0 +1,166 @@
+import uuid
```

Comment:
I'm trying to add strong typing to all redis implementations:

One nice to have is to add this file to `pyproject`'s `# beginning: stronger typing` section.

Reply:
Added the new redis service backend to the stronger typing block.